#!/usr/bin/env python
PACKAGE = "vmcl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("correct_distance", double_t, 0, "A double parameter", 2.0, 0.01, 10)

exit(gen.generate(PACKAGE, PACKAGE, "VMCL"))